packages/pharos-cli/src/utils/generation-utils.ts

```diff
@@ -68,3 +68,29 @@ export const addComponentToIndexFile = (
 
   console.log(colors.green(`\nPharos${nameOptions.titleCaseName} export added to ${filePath}`));
 };
+
+export const addComponentsToInitFile = (
+  currentDirectory: FilePath,
+  nameOptions: ComponentNameOptions
+): void => {
+  const componentName = `Pharos${nameOptions.titleCaseName}`;
+  const unitTestsFilePath: FilePath = `${currentDirectory}/packages/pharos/src/test/initComponents.ts`;
+  const storybookFilePath: FilePath = `${currentDirectory}/.storybook/initComponents.js`;
+
+  // import new component in list of imports
+  // and then register new component
+  [unitTestsFilePath, storybookFilePath].map((filePath: FilePath) => {
+    fs.readFile(filePath, 'utf8', function (_, data) {
+      const existingFileContent: string[] = data.split('} from');
+      const newImportStatement =
+        existingFileContent[0] + `  ${componentName}` + ',' + '\n' + '} from';
+      const updatedFileContent = newImportStatement + existingFileContent[1];
+      fs.writeFileSync(filePath, updatedFileContent);
+
+      const existingRegisterComponents: string[] = updatedFileContent.split(']);');
```

Comment:
This `split`ting feels brittle; if the contents of the file being manipulated changes substantially later, this operation could produce a bad file. Maybe it's a low-likelihood thing but something to think a bit about.

Reply:
@daneah  I agree with you. I also feel the same. However, I tried not to use this way of splittin,  but I couldn't come up with better way of doing this. Do you think of any better way like using pure typescript or such ? I would love to use that.

Reply:
The most robust way I know of to alter a file statically is to manipulate and validate an altered [abstract syntax tree](https://en.wikipedia.org/wiki/Abstract_syntax_tree). The TypeScript wiki hints at [some ways of doing that](https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API#re-printing-sections-of-a-typescript-file) using the TypeScript API. This is very possible overkill here; I won't necessarily encourage going down that road.

Reply:
@daneah  I am working on modifying source file using AST,  the links you shared are useful.  So, its just taking little longer than I expected. Hoping to get it done in next two days.

Reply:
I see that you mentioned, it would be overkill, but I wanna give it a shot. I liked that API a lot, it's interesting.

Reply:
@daneah  after spending good amount of time, I learned that using `ts-morph` package is helpful and it lets us do what we need in lesser code. I got stackblitz project with working function. I added details in README about how I used the script. Let me know your thoughts on this. More explicitly, I am looking for your suggestion on whether we can use `ts-morph` package in our  pharos site.

Reply:
Nice find! I think I'd welcome giving this a try, especially as a `devDependencies` in `pharos-cli`.

Reply:
Thanks, Dane. Will give it a shot today.

Reply:
@daneah I am facing this error when running `yarn add -D ts-morph` in `pharos-cli`. I am leaning towards installing the missing package, but wondering am I misunderstanding anything ? Can I get some help here ?

`error An unexpected error occurred: "expected workspace package to exist for \"@typescript-eslint/eslint-plugin\"".
`